app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java

```diff
@@ -141,11 +142,11 @@ private void setAddVehicleOnClickListener() {
             String city = edtCity.getText().toString();
             String state = edtState.getText().toString();
             String zipCode = edtZipCode.getText().toString();
-            getPlaceId(streetAddress, city, state);
+            getPlaceIdAndGeoPoint(streetAddress, city, state);
         });
     }
 
-    private void getPlaceId(String streetAddress, String city, String state) {
+    private void getPlaceIdAndGeoPoint(String streetAddress, String city, String state) {
```

Comment:
I wouldn't name this as getPlaceId b/c it is not returning anything. Instead, I would call it `initializeLocationInfor`